entries/p/plus500.com.json

```
@@ -0,0 +1,24 @@
+{
+  "Plus500": {
+    "domain": "plus500.com",
+    "additional-domains": [
+      "plus500.at",
+      "plus500.co.nz",
+      "plus500.co.uk",
+      "plus500.com.au",
+      "plus500.es",
+      "plus500.de",
+      "plus500.fr",
+      "plus500.it",
+      "plus500.nl",
+      "plus500.pt",
+      "plus500.ru"
+    ],
+    "tfa": [
+      "sms"
```

```suggestion
      "sms",
      "call"
```
The screenshot provided also mentions 2FA via phone call